lib/dialects/mysql/query-generator.js

```diff
@@ -265,6 +266,10 @@ var QueryGenerator = {
 
     if (attribute.references) {
       attribute = Utils.formatReferences(attribute);
+      if (options && options.context === 'addColumn' && options.attributeName) {
+        template += ', ADD FOREIGN KEY (' + this.quoteIdentifier(options.attributeName) + ')';
```

Comment:
I haven't checked yet, but does that match the naming we use elsewhere for foreign keys?